specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -682,6 +689,30 @@
       },
       "description": "Specifies the metric alert criteria for a single resource that has multiple metric criteria."
     },
+    "WebtestLocationAvailabilityCriteria": {
+      "x-ms-discriminator-value": "Microsoft.Azure.Monitor.WebtestLocationAvailabilityCriteria",
+      "type": "object",
+      "properties": {
+        "webTestId": {
+          "type": "string",
+          "description": "The web test Id."
```

Comment:
What is this - is this an ARM resource?

Reply:
Yes, it's an ARM resource (microsoft.insights/webtests)

Reply:
Thanks - would you mind updating the description slightly to reflect this then - "The web test resource Id"?

Reply:
Changed to "The Application Insights web test Id."